cluster/addons/fluentd-elasticsearch/fluentd-es-configmap.yaml

```diff
@@ -101,25 +111,19 @@ data:
     # CRI Log Example:
     # 2016-02-17T00:04:05.931087621Z stdout F [info:2016-02-16T16:04:05.930-08:00] Some log text here
     <source>
-      type tail
+      @id fluentd-containers.log
+      @type tail
       path /var/log/containers/*.log
-      pos_file /var/log/es-containers.log.pos
+      pos_file /var/log/fluentd-containers.log.pos
```

Comment:
is there a reason to change this?  could have adverse effects if people are upgrading.